tests/toranj/cli/test-020-net-diag-vendor-info.py

```diff
@@ -192,9 +193,31 @@
         verify(False)
 
 result = r2.cli('networkdiagnostic get', r1_rloc, MLE_COUNTERS_TLV)
-print(len(result) >= 1)
 verify(result[1].startswith("MLE Counters:"))
 
+# Test Non-preferred Channel Mask TLV
+
+result = r1.cli('networkdiagnostic nonprfchannelmask')
+verify(len(result) == 1)
+verify(int(result[0], 16) == 0)
+
+result = r2.cli('networkdiagnostic get', r1_rloc, NON_PRF_CHANNEL_MASK_TLV)
+verify(len(result) == 2)
+verify(result[1].startswith('Non-preferred Channel Mask:'))
+verify(int(result[1].split(':')[1].strip(), 16) == 0)
+
+mask = 1 << 26  # channel 26
+
+r1.cli('networkdiagnostic nonprfchannelmask', mask)
+result = r1.cli('networkdiagnostic nonprfchannelmask')
+verify(len(result) == 1)
+verify(int(result[0], 16) == mask)
+
+result = r2.cli('networkdiagnostic get', r1_rloc, NON_PRF_CHANNEL_MASK_TLV)
+verify(len(result) == 2)
+verify(result[1].startswith('Non-preferred Channel Mask:'))
+verify(int(result[1].split(':')[1].strip(), 16) == mask)
+
```

Comment:
(optional) could add here setting the mask back to `0` ?

Reply:
Added this step to the end.